charts/sonarqube-dce/README.md

```diff
@@ -230,7 +230,7 @@ The following table lists the configurable parameters of the SonarQube chart and
 | `searchNodes.replicaCount`                                | Replica count of the Search Nodes                                                     | `3`                                                                    |
 | `searchNodes.podDisruptionBudget`                         | PodDisruptionBudget for the Search Nodes                                              | `minAvailable: 2`                                                      |
 | `searchNodes.podDistributionBudget`                       | (DEPRECATED typo) PodDisruptionBudget for the Search Nodes                            | `minAvailable: 2`                                                      |
-| `searchNodes.securityContext.fsGroup`                     | Group applied to mounted directories/files on search nodes                            | `0`                                                                    |
+| `searchNodes.securityContext`                     | SecurityContext for the pod search nodes                            | [Restricted podSecurityStandard](#kubernetes---pod-security-standards)                                                                    |
```

Comment:
Did you run a Markdown formatter here?

Reply:
Good point. I run it on the whole readme and fixed other small inconsistencies.